cgmanifests/submodules/cgmanifest.json

```diff
@@ -255,7 +255,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "db78ac1d7716f56fc9f1b030b715f872f93964e4",
```

Comment:
this is just the manifest. but did you update the submodule itself?
i'm in the process of setting up a device with jetpack 4.6 to repro and validate.

Reply:
Updated the submodule as well